packages/stellar-dispatcher/src/index.js

```diff
@@ -0,0 +1,36 @@
+import uuid from 'uuid';
+import _ from 'lodash';
+
+const stoppersMap = {};
+let stellar;
+
+export function configure(_stellar) {
+    stellar = _stellar; // eslint-disable-line better-mutation/no-mutation
+}
+
+export const dispatch = ({ resource, path, method, channel, reactiveHandler, payload, options }) => {
+    if (method === 'unsubscribe') {
+        const stopper = stoppersMap[payload];
```

Comment:
payload as the key? pls comment what it really is

Reply:
I have no idea